addons/pinniped/config-controller/hack/run.sh

```diff
@@ -14,6 +17,16 @@ tag="dev"
 # tag="$(uuidgen)" # Uncomment to create random image every time
 image="harbor-repo.vmware.com/tkgiam/$(whoami)/pinniped-config-controller-manager:$tag"
 
+# be run to run ./generate.sh first to get current RBAC
```

Comment:
I could also just change this to always run `./generate.sh` before deployment to ensure the RBAC is always fresh.

Reply:
I like that idea!

Reply:
done